searchapp/static/student_view.js

```diff
@@ -1,27 +1,32 @@
 $(document).ready(function(){
 
 	// on selecting type of worksheet
-	$('#worksheetType').dropdown({
-		onChange: function (value, text, $selectedItem) {
-			if(value == 'test') {
-				$("#test-div").show();
-				$("#generic-div").hide();
-				$("#customized-div").hide();
-			} else if(value == 'generic') {
-				$("#test-div").hide();
-				$("#generic-div").show();
-				$("#customized-div").hide();
-			} else if(value == 'customized') {
-				$("#test-div").hide();
-				$("#generic-div").hide();
-				$("#customized-div").show();
-			}
-		},
-	});
+	function display_form_sections (value) {
+		if(value == 'test') {
+			$('#worksheetType').text('Test Paper');
+			$("#test-div").show();
+			$("#generic-div").hide();
+			$("#customized-div").hide();
+		} else if(value == 'generic') {
+			$('#worksheetType').text('Generic Paper');
+			$("#test-div").hide();
+			$("#generic-div").show();
+			$("#customized-div").hide();
+		} else if(value == 'customized') {
+			$('#worksheetType').text('Customized Paper');
+			$("#test-div").hide();
+			$("#generic-div").hide();
+			$("#customized-div").show();
+		}
+	}
 
 	// function called when submit button clicked
 	function submit_click(e) {
-		let worksheetType = $("#worksheetType").dropdown('get value');
+		let worksheet = $("#worksheetType").text();
```

Comment:
The worksheet type is the argument `value` in `display_form_sections` and that can be used instead of finding worksheet type again.